lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +36,137 @@ fields:
     example: |
       {
         "id": "SH-12508041"
-      }
\ No newline at end of file
+      }
+
+  - name: questionnaire
+    required: 'Yes'
+    type: Questionnaire
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire
+    description: The Questionnaire that defines and organizes the questions for which answers are being provided.
+    action: update
+    example: |
+      {
+        "questionnaire": "https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041"
+      }
+
+  - name: status
+    required: 'Yes'
+    type: code
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.status
+    description: The position of the questionnaire response within its overall lifecycle.
+    action: update
+    example: |
+      {
+        "status": "in-progress"
+      }
+    note: status must be 'in-progress'
+  
+  - name: subject
+    required: 'Yes'
+    type: Reference
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.subject
+    description: The subject of the questionnaire response. This could be a patient, organization, practitioner, device, etc. This is who/what the answers apply to, but is not necessarily the source of information.
```

Comment:
this has to be patient

Reply:
beebfd509abafe9ef2bdc1d95e03ff5037ae505d